sdk/tables/Azure.Data.Tables/src/ResponseFormat.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+
+namespace Azure.Data.Tables.Models
+{
+    /// <summary> The ResponseFormat. </summary>
```

Comment:
nit:  This doesn't offer much value in this form.   I'd recommend either removing it or filling it out.